src/clr-angular/layout/tabs/tab-content.ts

```diff
@@ -12,7 +12,14 @@ let nbTabContentComponents: number = 0;
 @Component({
   selector: 'clr-tab-content',
   template: `
+    <ng-template #tabContentProjectedRef>
+      <section [id]="tabContentId" role="tabpanel" [class.active]="active"
```

Comment:
If you add this, you need to remove the identical bindings on the host.

Reply:
Right... I also need to add the '[attr.aria-expanded]': 'active' that appears on the host, but not here. I'll do it.

Reply:
Can we include the `hidden` html attribute here?

Reply:
Added both for now.